EditorsDraft/index.html

```diff
@@ -1614,8 +1648,9 @@
       "type": "PostalAddress",
       "streetAddress": "North Parade Road",
       "addressLocality": "Bath",
-      "region": "Somerset",
-      "postalCode": "BA2 4ET"
+      "addressRegion": "Somerset",
+      "postalCode": "BA2 4ET",
+      "addressCountry": "UK"
```

Comment:
Should be "GB", the two-letter [ISO 3166-1 alpha-2 country code](http://en.wikipedia.org/wiki/ISO_3166-1).